tools/darwin/Support/Codesign.command

```diff
@@ -103,4 +98,21 @@ if ! [ -z "${CODE_SIGN_IDENTITY_FOR_ITEMS}" ]; then
       done
     fi
   fi
+elif [ ! "$MACOS" ]; then
+  # Do fake sign - needed for iOS >=5.1 and tvOS >=10.2 jailbroken devices
+  # See http://www.saurik.com/id/8
+  echo "Doing a fake sign using ldid for jailbroken devices"
+
+  "${LDID}" -S"${DARWIN_EMBEDDED_ENTITLEMENTS}" "${BUILT_PRODUCTS_DIR}/${EXECUTABLE_FOLDER_PATH}/${EXECUTABLE_NAME}"
+
+  # On tvOS we also need to fake sign the Top Shelf binary
+  if [ "$APPLETVOS" ]; then
+    if [ -d "${TOPSHELF_PATH}/_CodeSignature" ]; then
+      rm -r "${TOPSHELF_PATH}/_CodeSignature"
+    fi
+    if [ -f "${TOPSHELF_PATH}/embedded.mobileprovision" ]; then
```

Comment:
Is this actually needed? If it falls through to this codepath, it's already been deleted hasn't it?

Reply:
Do you mean that `_CodeSignature` and `embedded.mobileprovision` will always not exist in case `CODE_SIGN_IDENTITY_FOR_ITEMS` is empty?

Reply:
They are deleted at line 43 and 49 respectively. As it doesn't run codesign again in this script unless proper signing is provided, they won't be regenerated

Reply:
Line 43 and 49 delete the Kodi one but not the TopShelf one, no?

Reply:
Sorry, my bad. If that's the case, do a find and rm the result. Will do it in one hit. Doesn't need to be behind a platform check.
Does ldid actually care if it's there though?

Reply:
Like that --> last push?

ldid doesn't care, but on jailbroken devices those folder/files are not needed ; I just rm them to have a "clean" deb file without any signing stuff. Then the end user can directly see that he can (i) directly install it on jailbroken device or (ii) sign it with iOS App Signed.

Reply:
The only reason we delete them is for Dev use. A normal user (or a Jenkins build) will never have them anyway

Reply:
Oh yes, I always forget that Jenkins do not perform signing stuff, so you are right, those files should never be deb file produced by Jenkins.
So, to conclude, should we remove all the lines that delete `_CodeSignature` and `embedded.mobileprovision`?

I simply delete the "if appletvos" line at the end of the file.

Reply:
I moved the "delete" lines inside the "if ${CODE_SIGN_IDENTITY_FOR_ITEMS}", so that we don't care about `_CodeSignature` and `embedded.mobileprovision` files when we do not have signing stuff like Jenkins.

Also, I noticed a `libass` binary in Frameworks folder. ATM this one is not fake-signed and have "not signed at all". Not sure if it works on jailbroken devices ; how can I test this binary during Kodi runtime?